webclient/src/forms/LoginForm/LoginForm.tsx

```diff
@@ -1,46 +1,137 @@
 // eslint-disable-next-line
-import React from "react";
+import React, { Component, useCallback, useEffect, useState, useRef } from 'react';
 import { connect } from 'react-redux';
-import { Form, Field, reduxForm, change } from 'redux-form'
+import { Form, Field, reduxForm, change, FormSubmitHandler } from 'redux-form'
 
 import Button from '@material-ui/core/Button';
 
-import { InputField, KnownHosts } from 'components';
-// import { ServerDispatch } from "store";
-import { FormKey } from 'types';
+import { AuthenticationService } from 'api';
+import { CheckboxField, InputField, KnownHosts } from 'components';
+import { useAutoConnect } from 'hooks';
+import { HostDTO, SettingDTO } from 'services';
+import { FormKey, APP_USER } from 'types';
 
 import './LoginForm.css';
 
-const LoginForm = (props) => {
-  const { dispatch, handleSubmit } = props;
+const PASSWORD_LABEL = 'Password';
+const STORED_PASSWORD_LABEL = '* SAVED *';
```

Comment:
How about just "Password Saved"?

Reply:
I was planning to run this by Lee and the group as a follow up.